hypershift-operator/main.go

```diff
@@ -416,15 +418,23 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 	}
 
 	// If enabled, start controller to ensure UWM stack is enabled and configured
-	// to remotely write telemetry metrics
-	if opts.EnableUWMTelemetryRemoteWrite {
+	// to remotely write telemetry metrics. The UWM stack will be disabled if the
+	// cluster is IPv6.
+	mgmtNetworkConfig := manifests.OpenshiftNetworkConfiguration()
```

Comment:
Why are we disabling telemetry for all clusters having ipv6 in the management cluster?

Reply:
Because we are assuming that if the first entry of the serviceNetwork in the MGMT cluster is IPv6, the cluster is disconnected. As I discussed with @sjenning I don't know any other better way to infer if the cluster is disconnected or not. Any suggestion?